xds/internal/resolver/xds_resolver.go

```diff
@@ -184,6 +186,9 @@ type xdsResolver struct {
 	activeClusters map[string]*clusterInfo
 
 	curConfigSelector *configSelector
+
+	// A random number which uniquely identifies this client.
+	clientID uint64
```

Comment:
Switch to channelID? Also comment feels weird, "this client" in a resolver. I know the Client Conn owns this resolver with a 1:1 cardinality but this comment feels weird.

Reply:
Switched.  Not sure about "feels weird" or how to resolve that, but I changed the comment a bit.  In the future please be more specific about what's wrong and/or how you would fix it.

Reply:
Yeah sorry I meant this client is in reference to what? Your comment change works for me.